x-pack/test/functional/services/transform_ui/wizard.ts

```diff
@@ -75,6 +75,47 @@ export function TransformWizardProvider({ getService }: FtrProviderContext) {
       await testSubjects.existOrFail(selector);
     },
 
+    async parseTable(tableSubj: string) {
+      const table = await testSubjects.find(`~${tableSubj}`);
+      const $ = await table.parseDomContent();
+      const rows = [];
+
+      // For each row, get the content of each cell and
+      // add its values as an array to each row.
+      for (const tr of $.findTestSubjects(`~${tableSubj}Row`).toArray()) {
+        rows.push(
+          $(tr)
+            .find('.euiTableCellContent')
+            .toArray()
+            .map(cell =>
+              $(cell)
+                .text()
+                .trim()
+            )
+        );
+      }
+
+      return rows;
+    },
+
+    async assertTableResults(tableSubj: string, column: number, expectedColumnValues: string) {
```

Comment:
This method doesn't assert the whole table, so I would suggest to rename it to something like `assertPreviewTableColumnValues`.

Reply:
Fixed in 015aa86.